.ci_support/linux_64_numpy1.22python3.10.____cpython.yaml

```diff
@@ -13,19 +17,19 @@ cxx_compiler:
 cxx_compiler_version:
 - '9'
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
 fftw:
 - '3'
 fortran_compiler:
 - gfortran
 fortran_compiler_version:
 - '9'
 hdf5:
-- 1.12.2
+- 1.14.3
```

Comment:
The environment isn't solvable with this hdf5 pin:
```
Could not solve for environment specs
The following packages are incompatible
├─ hdf5 =1.14.3 * is requested and can be installed;
└─ libgdal =3.11 * is not installable because it requires
   └─ libgdal-netcdf =3.11.0 *, which requires
      └─ hdf5 >=1.14.6,<1.14.7.0a0 *, which conflicts with any installable versions previously reported.
```

but this pin is automatically rendered, so I'll have to look up how to change this setting...